_posts/2025-03-03-GPU-Accelerated-Vector-Search-OpenSearch-New-Frontier.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "GPU-accelerated vector search in Opensearch: A new frontier" 
+authors:
+  - navneev
+  - cnolet
+  - kshitizgupta21
+  - dylantong
+  - nwstephens
+  - vamshin
+date: 2025-03-03
+has_science_table: true
+categories:
+  - technical-posts
+meta_keywords: Vector Database, Vector Engine, k-NN plugin,  OpenSearch 3.0, k-nn search, Vector Search, Semantics Search, Index Build
```

Comment:
Please update the meta keywords and description with the following: 

_meta_keywords:_ vector database, GPU acceleration, OpenSearch vector engine, why use GPU acceleration, NVIDIA cuVS,  CAGRA algorithm, improved indexing speed, large-scale vector search

_meta_description:_ Discover how the GPU acceleration feature in OpenSearch dramatically improves vector search performance, reducing indexing time by 9.3x and costs by 3.75x using NVIDIA cuVS technology

Reply:
sure